reporters_db/data/reporters.json

```diff
@@ -6346,7 +6332,7 @@
     ],
     "Ct. Int'l Trade": [
         {
-            "cite_type": "federal",
+            "cite_type": "specialty",
```

Comment:
Can we figure out, the merger of these two.  In my view the CIT is created by the Constitution.  If thats the case, I assume CIT may have been referenced early on.  Can you find the first reference to it